packages/neutron-sdk/src/interchain_queries/v045/types.rs

```diff
@@ -403,3 +470,61 @@ impl KVReconstruct for Delegations {
         Ok(Delegations { delegations })
     }
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct UnbondingEntry {
+    pub balance: Uint128,
+    pub completion_time: Option<Timestamp>,
+    pub creation_height: u64,
+    pub initial_balance: Uint128,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct UnbondingResponse {
+    pub delegator_address: Addr,
+    pub validator_address: String,
+    pub entries: Vec<UnbondingEntry>,
+}
```

Comment:
Maybe add comments for each field and structure?
We have it everywhere else in the PR